src/Appwrite/Platform/Tasks/Restore.php

```diff
@@ -0,0 +1,342 @@
+<?php
+
+namespace Appwrite\Platform\Tasks;
+
+use Exception;
+use Utopia\App;
+use Utopia\DSN\DSN;
+use Utopia\Platform\Action;
+use Utopia\CLI\Console;
+use Utopia\Storage\Device;
+use Utopia\Storage\Device\DOSpaces;
+use Utopia\Storage\Device\Local;
+use Utopia\Validator\Boolean;
+use Utopia\Validator\Text;
+
+class Restore extends Action
+{
+    public const BACKUPS_PATH = '/backups';
+    public const DATADIR = '/var/lib/mysql';
+    public const DOWNLOAD_CHUNK_SIZE = 40 * 1024 * 1024; // Must be greater than 5MB;
+    public const VERSION = 'v1';
+    public const RETRY_DOWNLOAD = 5;
+    public const RETRY_PREPARE = 5;
+    public const RETRY_DECOMPRESS = 5;
+    public const RETRY_TAR = 5;
+    protected ?DOSpaces $s3 = null;
+    protected string $xtrabackupContainerId;
+    protected int $processors = 1;
+
+    public function __construct()
+    {
+        $this
+            ->desc('Restore a DB')
+            ->param('id', '', new Text(19), 'The backup identification')
```

Comment:
Let's add to the description an example of where to get the ID from

Reply:
Done 👍